portfolio/src/main/java/com/google/sps/usercomment/UserComments.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.usercomment;
+
+/**
+* Class object to define the structure of a user comment.
+*/
+public class UserComments {
+  public String user;
+  public String comment;
+  public long id;
+
+  /** Constructor(String, String, long)*/
+  public UserComments(String userInput, String commentInput, long entityID) {
+    user = userInput;
+    comment = commentInput;
+    id = entityID;
+  }
+}
```

Comment:
Please add GETers and SETers to this class, it is a good practice to have the attributes of a class as private and then access to them via GETers and SETers. That will help you to manipulate a comment outside this class.

Reply:
Thanks for the recommendation!